### PR TITLE
Issue 1929 - Document Policy Properties

### PR DESCRIPTION
**Pull request by sirotsinskuy**
Signed-off-by: Antonii Sirotsinskyi <asiro@softserveinc.com>

### CODE REVIEW

File: docs/policy.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Policy Properties
+
+There are built-in property names that can be used in the policies.
+anax will fill in the properties during the agreement negotiation process.
```
[the remark this answers is not in the document]
Comment (dabooz) [reply docs/policy.md]: This isn't quite resolved. You need to delete the sentence about anax filling in properties.

File: docs/built_in_policy.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+* for service policy
+
+**Name** | **Description** | **Possible values**
+----- | ----- | -----
+openhorizon.service.url | The unique name of the service (comes from `url` field of service definition) | `string` e.g. https://someOrg/someService
+openhorizon.service.name| The unique name of the service (comes from `url` field of service definition)| `string` e.g. MyService
+openhorizon.service.org| The multi-tenant org where the service is defined. The org defaults to the org of the node (comes from `org` field of service definition) | `string` e.g. MyOrg
+openhorizon.service.version| The version of a service using the same semantic version syntax (comes from `version` field of service definition)| `string` e.g. 1.1.1
+openhorizon.service.arch| The hardware architecture of the node this service can run on (comes from `url` field of service definition)| `string` e.g. amd64
```
Comment (dabooz) [inline docs/built_in_policy.md:31]: comes from the arch field of the service.

File: docs/built_in_policy.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+**Note:Provided properties (except for allowPrivileged) are read-only, the system will ignore updating of the node policy and changing any of the built-in properties*    
+
+* for service policy
+
+**Name** | **Description** | **Possible values**
+----- | ----- | -----
+openhorizon.service.url | The unique name of the service (comes from `url` field of service definition) | `string` e.g. https://someOrg/someService
+openhorizon.service.name| The unique name of the service (comes from `url` field of service definition)| `string` e.g. MyService
+openhorizon.service.org| The multi-tenant org where the service is defined. The org defaults to the org of the node (comes from `org` field of service definition) | `string` e.g. MyOrg
```
Comment (dabooz) [inline docs/built_in_policy.md:29]: Where did you get the info about this field defaulting? I dont think it does. It is always the org that the service is defined in.

**Review by dabooz — CHANGES_REQUESTED** [review]

File: docs/built_in_policy.md (hunk 3 above)
Comment (sirotsinskuy) [reply docs/built_in_policy.md]: Do we need that sentence to remain in this doc?

Comment (dabooz) [reply docs/built_in_policy.md]: I think the way you have it worded right now is fine.

**Review by dabooz — APPROVED** [review]

**Pull request merged**